app/services/mapping/governmentagencygoodsitem/GovernmentProcedureBuilder.scala

```diff
@@ -15,13 +15,32 @@
  */
 
 package services.mapping.governmentagencygoodsitem
+import forms.declaration.ProcedureCodes
+import services.cache.ExportItem
+import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment.GovernmentAgencyGoodsItem.GovernmentProcedure
 import wco.datamodel.wco.declaration_ds.dms._2.{GovernmentProcedureCurrentCodeType, GovernmentProcedurePreviousCodeType}
 
 import scala.collection.JavaConverters._
 
 object GovernmentProcedureBuilder {
```

Comment:
Similar to above, can we make this a class and inject the dependent builders

Reply:
this is TODO as well as the PR title suggested this is PART1